website/pages/docs/post-processors/vagrant.mdx

```diff
@@ -135,6 +199,16 @@ Specify overrides within the `override` configuration by provider name:
 }
 ```
 
+</Tab>
+<Tab heading="HCL2">
+
+```hcl
+## This feature is not implemented in HCL.
```

Comment:
Am I right that "builder specific overrides" aren't really a thing in HCL? https://www.packer.io/docs/templates/provisioners#build-specific-overrides